<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorScene</class>
 <widget class="QMainWindow" name="EditorScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="showTab"/>
    <addaction name="startPlay"/>
    <addaction name="toTitle"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="showTab">
   <property name="text">
    <string>部件栏</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;显示部件栏&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="startPlay">
   <property name="text">
    <string>游玩</string>
   </property>
   <property name="toolTip">
    <string>开始玩耍</string>
   </property>
  </action>
  <action name="toTitle">
   <property name="text">
    <string>返回标题</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
